### ISSUE TITLE
External IDE - Android Studio does not work

### ISSUE THREAD

**Issue by zendorx** [defold/defold#8982]
<img width="594" alt="screenshot" src="https://github.com/defold/defold/assets/909905/63b12a78-a132-40b8-854e-9dc6d2ae2a2a">

If set android studio path on mac os Defold Editor wont open this path. I think it is because of space in "Android Studio" name. I have tested it with PyCharm and it works ok

1. Install Android Studio
2. Set path for external editor "/Applications/Android Studio.app/Contents/MacOS/studio"
3. Try to open script file
4. Nothing happens
5. Install PyCharm
6. Set path for external editor "/Applications/PyCharm CE.app/Contents/MacOS/pycharm"
7. Try to open script file
8. Everything works fine

**Defold version (REQUIRED):**
  - Version 1.8.0

MAC OS

**Comment by britzl** [comment]
Can you open individual files like that or does it have to be an Android Studio Project? Have a look at some of the answers here: https://stackoverflow.com/questions/35363546/open-androidstudio-project-from-command-line-on-osx

**Comment by zendorx** [comment]
I open file like that from terminal
`/Applications/Android\ Studio.app/Contents/MacOS/studio . --line 100 "/Users/ds/rpg/defold/rpg/im/debug_utils.lua"`

**Comment by britzl** [comment]
It turns out that /Applications/Android\ Studio.app/Contents/MacOS/studio is a Java class file, not an executable. It loads with the help of the Info.plist where classpath and other details are specified. I'm not sure how we can do the same from the Defold editor.

**Comment by bjconlan** [comment]
I was thinking that perhaps there needs to be an `xdg-open`, `open` or `start` based invocation but based on[ this stack overflow answer](https://stackoverflow.com/questions/35363546/open-androidstudio-project-from-command-line-on-osx), it might be simply overcome by letting the ide create the appropriate command line binding and use that from defold?

**Comment by britzl** [comment]
> it might be simply overcome by letting the ide create the appropriate command line binding and use that from defold?

Yes, perhaps that is sufficient. On macOS I get this suggestion when selecting "Configure Command-Line Launcher":

<img width="473" alt="Screenshot 2024-08-07 at 08 04 06" src="https://github.com/user-attachments/assets/35e84363-c02f-4288-a9d8-a845ef9d42da">

You could try adding the suggested path to your system path and see if that's enough.
